app/kubemci/cmd/create.go

```diff
@@ -118,6 +124,8 @@ func addCreateFlags(cmd *cobra.Command, options *CreateOptions) error {
 	// TODO(nikhiljindal): Add a short flag "-p" if it seems useful.
 	cmd.Flags().StringVarP(&options.GCPProject, "gcp-project", "", options.GCPProject, "[required] name of the gcp project")
 	cmd.Flags().BoolVarP(&options.ForceUpdate, "force", "f", options.ForceUpdate, "[optional] overwrite existing settings if they are different")
+	cmd.Flags().StringVarP(&options.Namespace, "namespace", "n", options.Namespace, "[optional] namespace for the ingress unless provided by ingress spec")
```

Comment:
The description indicates that namespace flag is ignored if ingress spec provides a namespace.
That is not true? Should probably also mention in the description what happens if it conflicts with namespace in ingress spec.
(same comment for static ip)